Octokit/Models/Response/Organization.cs

```diff
@@ -10,12 +10,13 @@ public class Organization : Account
     {
         public Organization() { }
 
-        public Organization(string avatarUrl, string bio, string blog, int collaborators, string company, DateTimeOffset createdAt, int diskUsage, string email, int followers, int following, bool? hireable, string htmlUrl, int totalPrivateRepos, int id, string nodeId, string location, string login, string name, int ownedPrivateRepos, Plan plan, int privateGists, int publicGists, int publicRepos, string url, string billingAddress, string defaultRepositoryPermission, bool membersCanCreateRepositories)
+        public Organization(string avatarUrl, string bio, string blog, int collaborators, string company, DateTimeOffset createdAt, int diskUsage, string email, int followers, int following, bool? hireable, string htmlUrl, int totalPrivateRepos, int id, string nodeId, string location, string login, string name, int ownedPrivateRepos, Plan plan, int privateGists, int publicGists, int publicRepos, string url, string billingAddress, string defaultRepositoryPermission, bool membersCanCreateRepositories, string membersAllowedRepositoryCreationType)
```

Comment:
`defaultRepositoryPermission` and `membersAllowedRepositoryCreationType` are `string`s in the `ctor` rather than the enum types.  Although this does compile, we should take the enum type in the ctor for consistency/clarity